common/info.yaml

```
@@ -1292,6 +1292,128 @@ description: |
       Number and value of presets undefined and may vary by model.
       Useful for very basic integrations where you have just a single push button.
 
+  ## Color
+
+  The Color feature is available on full color lighting products.
+
+  The Color feature requires the Light and Brightness features.
+  The ColorTemp feature is usually not available at the same time as Color.
+
+  When `color = 1`, the values in `rgb` and `hsv` 
+  are active and specify the color of light from the device.
+
+  Whenever `SetBrightness` or other brightness-changing actions are called,
+  `rgb` and `hsv` values are updated to reflect the change. For example, if the 
+  rgb values are (128, 0, 0) with brightness 50, and brightness is increased
+  to 100, then rgb values are updated to (255, 0, 0).
+
+  **Note regarding the value (`v`) component of HSV colorspace:**
+  An RGB (or RGBW) LED has an inherent range of colors which it can produce.
+  This is known as the device's *gamut*. In RGB- (or RGBW-) space the shape
+  of the gamut is a cube (or hypercube). However, when this gamut is transformed
+  into HSV space, it becomes tent-shaped (a cylinder with a cone on top).
+  Intitively, what is going on is that when producing a pure color, 
```

Did you mean `Intuitively`?
```suggestion
  Intuitively, what is going on is that when producing a pure color, 
```